src/pages/index.js

```diff
@@ -119,45 +181,129 @@ function closeModalEscape(evt) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  const submitButton = evt.submitter
+  submitButton.textContent = "Saving...."
+
+  api
+  .editUserInfo({ name: editModalNameInput.value,
+    about: editModalDescriptionInput.value
+  })
+  .then((userData) => {
+
+    profileName.textContent = userData.name;
+    profileDescription.textContent = userData.about;
+    closeModal(editModal);
+  })
+  .catch(console.error)
+  .finally(() => {
+   submitButton.textContent = "Save";''
```

Comment:
Please, reuse `SetButtonText` for changing the button text, not to duplicate the same code